service-tests/codesponsor.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'codesponsor', title: 'Code Sponsor' });
+module.exports = t;
+
+t.create('earnings on actual repo')
+  .get('/hopsoft/bg.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('sponsor'),
+    value: Joi.string().regex(/^[0-9]+(\.[0-9]+)? BTC/)
+  }));
+
+t.create('display bitcoin amount')
+  .get('/hopsoft/bg.json')
+  .intercept(nock => nock('https://app.codesponsor.io')
+    .get('/shield/hopsoft/bg')
+    .reply(200, {
+      bitcoin: 0.004359,
```

Comment:
This test is going to start failing if this value ever changes... Better to do`Joi.number()` here.

Reply:
I will fix it. Thanks you

Reply:
@Daniel15 This is now fixed. Thanks!